### PR TITLE
Walletview

### PR DESCRIPTION
**Pull request by marcosrdz**
Modified for shared element transition 

## Summary by CodeRabbit

- **New Features**
  - Introduced a `WalletView` component for enhanced cryptocurrency wallet management.
  - Added support for multiple display types and improved user interactions including balance visibility and currency preferences.
  - New `CommonToolTipActions` for wallet management, improving user experience with additional tooltip actions.
  
- **Bug Fixes**
  - Enhanced error handling for wallet unit changes and interactions.

- **Refactor**
  - Streamlined navigation options using a dynamic helper function for better flexibility.
  - Consolidated wallet presentation logic by replacing old components with the new `WalletView`.

- **Chores**
  - Removed unused navigation parameters to simplify the navigation structure.

### CODE REVIEW

**Review by colorado392 — APPROVED** [review]

**Review by colorado392 — APPROVED** [review]

**Review by colorado392 — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

File: screen/wallets/transactions.js [hunk 1]
```diff
@@ -462,15 +463,30 @@ const WalletTransactions = ({ navigation }) => {
...
+            // Ensure unit is defined
+            InteractionManager.runAfterInteractions(async () => {
+              // Update the wallet's preferred balance unit
+              wallet.preferredBalanceUnit = unit;
+
+              // Update the UI to reflect the new unit
+              setItemPriceUnit(unit);
+
+              // Save the changes to disk
+              await saveToDisk();
```
Comment (Overtorment) [inline screen/wallets/transactions.js:484]: are those chatgpt comments..?

**Review by Overtorment — APPROVED** [review]
lgtm but check out coderabbit remarks. are those legit?

**Comment by marcosrdz** [comment]
this PR is invalid for what the intention was. Sadly. Closing.

**Pull request closed without merging**